config/config-default.json

```diff
@@ -11,7 +12,7 @@
     "files-scs:config/presets/ldp/request-parser.json",
     "files-scs:config/presets/logging.json",
     "files-scs:config/presets/setup.json",
-    "files-scs:config/presets/storage.json",
+    "files-scs:config/presets/storage-inMemory.json",
```

Comment:
inconsistent naming (others have… hyphens and underscores, so still inconsistent, but let's at least not make it worse 😄)

Reply:
Both `storage-inmemory` and `storage-in-memory` feel wrong though 😄 . This does remind me that I was going to rename the one with the underscore but didn't yet to not have merge issues with the previous PRs.

Reply:
`storage-memory`